src/services/conversionRates/getConversionRatesService.js

```diff
@@ -283,10 +285,13 @@ const getConversionRates = async (app, requestedDate, requestedSymbol = 'ETH') =
 
   const tokens = app.get('activeTokenWhitelist');
   let coingeckoId = '';
-
+  let equivalentSymbol = undefined;
   tokens.forEach(token => {
     if (token.symbol === requestedSymbol) {
+      //This field needed for PAN currency
       coingeckoId = token.coingeckoid;
+
+      equivalentSymbol = token.equivalentSymbol
```

Comment:
rename the field to more self descriptive like `rateSymbol` or `rateEqSymbol`

Reply:
You are right it's better